suites/upgrade/infernalis/older/1-install/latest_infernalis_release.yaml

```diff
@@ -0,0 +1,15 @@
+meta:
+   - desc: |
+      install ceph/infernalis
+      file system - xfs
```

Comment:
xfs is set in the overrides  in 0-start, so it can be removed from both of these 1-install files